src/codegen/utilities/properties/attribute.py

```diff
@@ -0,0 +1,281 @@
+class attribute():
```

Comment:
```suggestion
class Attribute():
```
Adopt [coding conventions](https://github.com/ni/python-styleguide/blob/main/docs/Coding-Conventions.md)

Reply:
Updated